env/index.js

```diff
@@ -1,9 +1,6 @@
-'use strict';
```

Comment:
We don't refuse actually, we do need.
Just unnecessary inside modules due to ESLint Airbnb style guide.